shared/chat/blocking/invitation-to-block.tsx

```diff
@@ -43,7 +43,7 @@ const BlockButtons = (props: Props) => {
         small={true}
         conversationIDKey={props.conversationID}
         toMany={others.length > 0 || !!team}
-        style={styles.button}
+        style={Styles.collapseStyles([!Styles.isMobile && styles.button])}
```

Comment:
```suggestion
        style={Styles.isMobile ? undefined : styles.button}
```

Reply:
I decided to use a different styles instead. 👍